scripts/zones/Southern_San_dOria_[S]/DefaultActions.lua

```diff
@@ -1,6 +1,70 @@
 local ID = require("scripts/zones/Southern_San_dOria_[S]/IDs")
 
 return {
+    ['_6eo'] = { text = ID.text.GATE_IS_LOCKED }, -- Yes, even though it's a door.
+    ['Achtelle'] = { text = ID.text.ACHTELLE_FROM_ADOULIN },
+    ['Adjutant'] = { text = ID.text.TOO_BUSY },
+    ['Aissaville'] = { event = 605 },
+    ['Andagge'] = { event = 53 },
```

Comment:
Would it be possible to format this list?  Probably something that can wait until we have all necessary mobs in it, but it looks like that may be close as well.

Reply:
There are still more possible Freelance NPCs that either don't exist in the DB correctly yet or I didn't see up on retail to add.